modules/ROOT/pages/dataweave-selecting-types.adoc

```diff
@@ -0,0 +1,206 @@
+= Selecting Types
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Using the dot selector (`.`) over DataWeave xref:dataweave-type-system.adoc#composite-types[types] enables you to declare new types from existing ones. Combined with the xref:dataweave-selectors-reusing-types.adoc[module loader], DataWeave can also load and translate declarations from a custom module file into DataWeave type directives that can be accessed in the same way as types from any other DataWeave module.
```

Comment:
I assume "over" is correct here (I'm not familiar with these programming terms).